comp/metadata/inventorychecks/inventorychecksimpl/inventorychecks.go

```diff
@@ -52,11 +52,12 @@ type checksMetadata map[string][]metadata
 
 // Payload handles the JSON unmarshalling of the metadata payload
 type Payload struct {
-	Hostname     string                `json:"hostname"`
-	Timestamp    int64                 `json:"timestamp"`
-	Metadata     map[string][]metadata `json:"check_metadata"`
-	LogsMetadata map[string][]metadata `json:"logs_metadata"`
-	UUID         string                `json:"uuid"`
+	Hostname     string                   `json:"hostname"`
+	Timestamp    int64                    `json:"timestamp"`
+	Metadata     map[string][]metadata    `json:"check_metadata"`
+	LogsMetadata map[string][]metadata    `json:"logs_metadata"`
+	CheckResults []map[string]interface{} `json:"check_results"`
```

Comment:
We should embed the result of each checks within `check_metadata` instead of having 2 different maps about the same thing.

Could you also update the README of the module which we use as documentation ?